sw/host/opentitanlib/src/tpm/driver.rs

```diff
@@ -187,14 +190,43 @@ pub trait Driver {
     }
 }
 
+type GpioPinAndMonitoring = (Rc<dyn gpio::GpioPin>, Rc<dyn gpio::GpioMonitoring>);
+
+fn wait_for_gsc_ready(gsc_ready_pin: &Option<GpioPinAndMonitoring>) -> Result<()> {
+    let Some((gsc_ready_pin, monitoring)) = gsc_ready_pin else {
+        return Ok(());
+    };
+    let start_time = SystemTime::now();
```

Comment:
Should we use `std::time::{Instant,Duration}` instead of SystemTime?

Reply:
After reading the documentation of `std::time::Instant` it appears that it exactly fits the bill.  I had initially confused myself by assuming that `Instant` shared characteristics with the Java class of the same name.